README.md

```diff
@@ -249,7 +251,8 @@ cargo run --example $name
 [`Layer`]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html
 [`tracing` target]: https://docs.rs/tracing/latest/tracing/struct.Metadata.html
 [`TRACE` level]: https://docs.rs/tracing/latest/tracing/struct.Level.html#associatedconstant.TRACE
-[build]: https://tokio-console.netlify.app/console_subscriber/fn.build.html
+[builder]: https://tokio-console.netlify.app/console_subscriber/struct.builder.html
 [init]: https://tokio-console.netlify.app/console_subscriber/fn.init.html
```

Comment:
these linked to the Netlify documentation for the `main` branch because the `console-subscriber` crate had not been published to creates.io at the time the README was written. now, though, `console-subscriber` is published, so we should change these to link to docs.rs instead:
```suggestion
[builder]: https:/docs.rs/console-subscriber/latest/console_subscriber/struct.builder.html
[init]: https:/docs.rs/console-subscriber/latest/console_subscriber/fn.init.html
```